registry.go

```diff
@@ -0,0 +1,220 @@
+package gitlab
+
+import (
+	"fmt"
+	"net/url"
+	"time"
+)
+
+// RegistryService handles communication with the container registry related methods
+// of the GitLab API.
+//
+// GitLab API docs: https://docs.gitlab.com/ee/api/container_registry.html
+type RegistryService struct {
+	client *Client
+}
+
+// RegistryRepository represents a GitLab content registry repository.
+//
+// GitLab API docs: https://docs.gitlab.com/ee/api/container_registry.html
+type RegistryRepository struct {
+	ID        int        `json:"id"`
+	Name      string     `json:"name"`
+	Path      string     `json:"path"`
+	Location  string     `json:"location"`
+	CreatedAt *time.Time `json:"created_at,omitempty"`
```

Comment:
`omitempty` can be omitted here as this struct is only used to unmarshal the response.